interfaces/ip_router/v0/README.md

```diff
@@ -38,6 +38,7 @@ Both the Requirer and the Provider need to adhere to criteria to be considered c
 ```yaml
 requirer:
   app: {
+    "network-name: "<network-name-1>"
```

Comment:
This section is an example so please use example values instead of placeholders

Reply:
Updated